src/main/kotlin/racing/Cars.kt

```diff
@@ -1,13 +1,30 @@
 package racing
 
-class Cars(private val carNumber: Int) {
+class Cars {
+    val carList: List<Car>
+    constructor(carList: List<Car>) {
+        this.carList = carList
+    }
```

Comment:
이 생성자를 주 생성자로 만들어보는 건 어떨까요?
```suggestion
class Cars(val carList: List<Car>) {
...
```

Reply:
훨씬 깔끔하네요 : )